src/cmd_definitions.c

```diff
@@ -23,6 +24,11 @@ const cmd_definition *cmd_get_definition(char *cmd_name) {
   return NULL;
 }
 
+void cmd_help(char **arg_list, size_t arg_count, char *res_msg) {
+  sceAppMgrDestroyOtherApp();
```

Comment:
Why do you introduce this side effect? It's probably unwanted.

Reply:
oops  
will fix that